Tests/SwiftGenKitTests/TestsHelper.swift

```diff
@@ -155,6 +155,40 @@ func convertToString(_ value: Any) -> String? {
   }
 }
 
+func convertToDictionary(_ value: Any) -> [String: Any]? {
+  switch value {
+  case let value as [String: Any]:
+    return value
+  case let value as NSObject:
+    return convertNSObjectToDictionary(value)
+  default:
+    return nil
+  }
+}
+
+func convertNSObjectToDictionary(_ object: NSObject) -> [String: Any] {
+  var result: [(String, Any)] = []
+  var mirror: Mirror? = Mirror(reflecting: object)
+
+  repeat {
+    // ensure we can property handle lazy variables
+    // we use `object.value(forKey:)` to force load lazy variables
+    result += mirror?.children
+      .compactMap { $0.label?.deletingPrefix("$__lazy_storage_$_") }
```

Comment:
So apparently at runtime when mirroring, labels for lazy vars get changed from `metadata` to `$__lazy_storage_$_metadata`...

Reply:
This should be part of the comment imho

```
// At runtime when mirroring, labels for lazy vars get the "$__lazy_storage_$_" prefix
// so we need to delete the prefix to handle those lazy var cases.
// We use `object.value(forKey:) to force load lazy variables because `someMirrorChild.value`
// evaluates to `nil` for lazy variables and doesn't trigger the lazy var evaluation.
```